core/templates/components/question-difficulty-selector/question-difficulty-selector.directive.html

```diff
@@ -1,14 +1,10 @@
 <md-radio-group ng-model="skillWithDifficulty._difficulty">
   <md-radio-button ng-repeat="rubric in getSkillIdToRubricsObject()[skillWithDifficulty.getId()]" value="<[availableDifficultyValues[$index]]>">
     <strong> <[rubric.getDifficulty()]> </strong>
-    <md-list>
-      <md-list-item ng-repeat="explanation in rubric.getExplanations()">
```

Comment:
Why is this removed? We have multiple explanations per rubric now and this directive is used elsewhere as well.

Reply:
The changes in #9148 caused the issues outlined [here](https://github.com/oppia/oppia/issues/9417#issuecomment-637717102), and the list elements were causing the formatting issues. I'd imagine since the topics and skills dashboard uses the same `select-skill-and-difficulty-modal` which uses the new `question-difficulty-selector` directive, the formatting issues are reflected in the topics and skills dashboard as well. Since this change is urgently needed for a hotfix, I opted for the simpler reversion here.

Reply:
The formatting is like that because there are multiple elements, which is required.
I think it is used when creating a question and when changing the difficulty of a question in questions-list, so this change here shouldn't be done as multiple explanations per rubric is something we support and this would break the editors.
You can try fixing the formatting here, or if this is urgent, in a follow up PR and just add the 'include' below in this PR.